bin/install_icons_and_menus.sh

```diff
@@ -462,6 +462,23 @@ EOF
 cp -a "/usr/share/applications/$WORKSHOP_INSTALL_FILE" "$USER_HOME/Desktop/"
 chown $USER_NAME.$USER_NAME "$USER_HOME/Desktop/$WORKSHOP_INSTALL_FILE"
 
+##### Setup INSPIRE installation icon
+WORKSHOP_INSTALL_FILE="inspire.desktop"
```

Comment:
This should be named INSPIRE_INSTALL_FILE

Reply:
Indeed so. Thanks.

Reply:
I will commit this fix after we merge